R/toolSplitSubtype.R

```diff
@@ -0,0 +1,32 @@
+#' Tool: SplitSubtype
+#' 
+#' This function can split a subtype string into smaller entities based on 
+#' a given separator and check the these entities exist in a reference list
```

Comment:
"check the these entities", probably: "check if or whether these entities exist"